client/src/balrogclient/api.py

```diff
@@ -83,6 +85,45 @@ def __call__(self, r):
         return r
 
 
+def get_balrog_session(auth0_secrets, session=None):
+    if not session:
+        session = requests.Session()
+
+    access_token = _get_auth0_token(auth0_secrets, session=session)
+    session.auth = BearerAuth(access_token)
+    return session
+
+
+def balrog_request(session, *args, **kwargs):
+    if len(args) != 2:
+        raise ValueError("args must have 2 values: (method, url)")
```

Comment:
If there's a hard limit then why *args and not just listing those two arguments in the function declaration? Then below it could be `session.request(method, url, **kwargs)`

Reply:
Whoops! This is clearly wrong on second look. I think your idea makes sense, but I _think_ it would probably be best to make the signature `def balrog_request(session, method, url, *args, **kwargs)`, in order to catch any other positional arguments as well? What do you think?

Reply:
Yeah, that sounds good.